modules/configuring-registry-storage-nutanix.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
+// * registry/configuring_registry_storage/configuring-registry-storage-nutanix.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-registry-storage-nutanix_{context}"]
+= Configuring registry storage for Nutanix
+
+As a cluster administrator, following installation you must configure your registry to use storage.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* A cluster on Nutanix. 
+* You have provisioned persistent storage for your cluster, such as {rh-storage-first}.
++
+[IMPORTANT]
+====
+{product-title} supports `ReadWriteOnce` access for image registry storage when you have only one replica. `ReadWriteOnce` access also requires that the registry uses the `Recreate` rollout strategy. To deploy an image registry that supports high availability with two or more replicas, `ReadWriteMany` access is required.
+====
++
+* You must have 100 Gi capacity.
+
+.Procedure
+
+. To configure your registry to use storage, change the `spec.storage.pvc` in the `configs.imageregistry/cluster` resource.
++
+[NOTE]
+====
+When using shared storage, review your security settings to prevent outside access.
```

Comment:
```suggestion
When you use shared storage, review your security settings to prevent outside access.
```
From IBM Style [Word Usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage):
>To avoid ambiguity, replace _using_ with either _by using_ or _that uses_.

To refer to something using one thing or another is often ambiguous. For example, the sentence _Back up the server using ABC_ might mean _Back up the server by using ABC_ or _Back up the server that uses ABC_. This advice also applies to gerunds other than using. For more information, see the style and grammar tips in the [Global audiences](https://www.ibm.com/docs/en/ibm_style/global-audiences.html) topic in IBM Style.